### ISSUE TITLE
Magic with dates, and url encoded +

### ISSUE THREAD

**Issue by RWOverdijk** [balderdashy/sails-mongo#343]
The following code gives Date a string, that's nearly identical except for the `+`.

```javascript
> new Date('Tue Sep 29 2015 23:05:23 GMT+0200 (CEST)')
Tue Sep 29 2015 23:05:23 GMT+0200 (CEST)
> new Date('Tue Sep 29 2015 23:05:23 GMT 0200 (CEST)')
Wed Sep 30 2015 01:05:23 GMT+0200 (CEST)
```

Currently, sails allows comparing dates for find(), but this only works for blueprints because of one simple bug: The plus is perceived as a space when sent along in a request url.

This means that a create would store the date wrongly, causing you to only be able to query dates using the blueprints, as they contain the same flaw.

I suggest fixing this at the core, in the criteria parser. If it's a date, try to fix it by putting back the + (if needed)

**Comment by sailsbot** [comment]
Thanks for posting, @RWOverdijk.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by RWOverdijk** [comment]
sails bot is annoying.
